src/modals/Transfer.tsx

```diff
@@ -63,12 +59,11 @@ function Transfer() {
         if (account) {
             setTransferLoading(true);
             try {
-                const amount = await Utils.toWei(transfer.amount.toString());
-                await Utils.sendTransaction({
+                await provider!.getSigner().sendTransaction({
                     from: account, // currentSmartWallet.address,
                     to: transfer.address,
-                    value: amount,
-                    data: '0x'
+                    value: transfer.amount,
+                    gasPrice: '60000000'
```

Comment:
Shouldn't we retrieve this value either from the configuration or from the provider?